app/views/unit_preferences/_preference_list.html.erb

```diff
@@ -5,7 +5,7 @@
         <tr>
           <th class="header_th">Name</th>
           <th class="header_th">Description</th>
-          <th class="header_th"></th>
+          <td></td>
```

Comment:
Why empty tag ?

Reply:
For the same reasons as above:
![Screenshot 2023-05-22 at 4 00 35 PM](https://github.com/lsa-mis/lsa_ride_share/assets/56089936/22926133-e1a5-4b57-a78d-8b018f7bf036)

I can put something into the header, 'Action', for example, then I'll use <th>Action</th>. But if there is no header title it's recommended to use <th></th> for accessibility.